test/general-tests/declare-v2-test/hardhat.config.ts

```diff
@@ -2,7 +2,9 @@ import "@shardlabs/starknet-hardhat-plugin";
 
 module.exports = {
     starknet: {
-        network: process.env.NETWORK
+        venv: "active",
+        network: process.env.NETWORK,
+        cairo1BinDir: process.env.CAIRO_1_COMPILER_DIR
```

Comment:
Why change this?

Reply:
I'm not sure. The compiler here is just a testing utility, we don't have to test a testing utility.

Reply:
https://github.com/0xSpaceShard/starknet-hardhat-plugin/pull/399/commits/c8f58ffedb82e9a1a794c733d17d31c40e80d2a7